Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/COREX12RealtimeTransforms.java

```diff
@@ -0,0 +1,342 @@
+/*
```

Comment:
Please use the latest license from here https://connectopensource.atlassian.net/wiki/display/CONNECTWIKI/CONNECT+License+Agreement.